.flowconfig

```diff
@@ -10,4 +10,18 @@
 <PROJECT_ROOT>/node_modules/editions
 
 [options]
-server.max_workers=1
\ No newline at end of file
+server.max_workers=1
+
+# $FlowFixMe
+# This suppression hould be used to suppress an issue that requires additional
+# effort to be resolved. That effort should be recorded in our issue tracking
+# system for follow-up. Usually, this suppression is added by tooling during
+# flow upgrades and should not be used directly by developers. If it is
+# necessary, make sure to raise the corresponding issue and add the issue
+# number to a TODO comment associated with the suppression.
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe
+
+# $FlowExpectError
```

Comment:
TIL: This is a good way to solve that

Reply:
Yeah, if we also run flow where warnings are errors, then this will work even better. I shied away from that just yet, but I think it's not a bad idea.